bin/ream/src/main.rs

```diff
@@ -32,15 +33,24 @@ async fn main() {
 
             let bootnodes = Bootnodes::new(config.network.network);
 
+            let beacon_chain = Arc::new(BeaconChain::mock_init());
+            let server_config = ServerConfig::from_args(
+                config.http_port,
+                config.http_address,
+                config.http_allow_origin,
+            );
+
+            let http_future = start_server(beacon_chain, server_config);
+
             let discv5_config = discv5::ConfigBuilder::new(discv5::ListenConfig::from_ip(
                 Ipv4Addr::UNSPECIFIED.into(),
-                8080,
+                config.discv_listen_port,
             ))
             .build();
             let binding = NetworkConfig {
```

Comment:
```suggestion
            let bootnodes = Bootnodes::new(config.network.network);
            let binding = NetworkConfig {
```

Could the bootnodes line be moved here